### ISSUE TITLE
Tailscale Stuck "Loading" on v1.44.0 & macOS 10.13.6

### ISSUE THREAD

**Issue by sumocomputers** [tailscale/tailscale#8694]
### What is the issue?

Tailscale is Stuck "Loading". The VPN portion connects, but you cannot login or get a Tailscale IP address.

Version 1.44.0 was working with macOS 10.13.6 in the past. Internet is working OK. Here is what I tried so far:

-Reset Tailscale to defaults, no change.
-Uninstalled and reinstalled Tailscale, no change.
-Rebooted computer, no change.
-Disconnected VPN manually, no change

![Screen Shot 2023-07-23 at 3 41 37 PM](https://github.com/tailscale/tailscale/assets/656453/6fb603c6-5886-4a18-9bd8-2e9e202f1836)

### OS

macOS

### OS version

10.13.6

### Tailscale version

1.44.0

### Bug report

BUG-6f889d89a53355dd220a3a7f70aa0b992f5611d6ea49c3f3b7dbdbccb43a63db-20230723225246Z-179b001e8a60aeaf

**Comment by sumocomputers** [comment]
I checked my DNS servers (1.1.1.1 & 9.9.9.9), and they were fine, yet I could not ping controlplane.tailscale.com.

I simply removed the macOS DNS entries, re-added them exactly as they were, and now I can ping controlplane.tailscale.com and Tailscale now works.

**Comment by Fut1l3** [comment]
I encountered this problem on my Apple Mac and chose to uninstall and reinstall but with the same outcome.  Unlike Linux it is jot so easy to clean up when uninstalling.  I had been experimenting with Exit Nodes and the Allow local Network Access was still enabled after switching it back to None.  Maybe a red herring but worth including the observation.  I was able to get the service operational by disabling my PIA VPN which also uses Wireguard.  I think there is generally a possible conflict as commercial VPN's are expected to work alongside a VPN that is also using Wireguard.
